mmtrack/models/aggregators/selsa_aggregator.py

```diff
@@ -68,10 +71,10 @@ def forward(self, x, ref_x):
         ref_x_new = self.ref_fc(ref_x)
         # [num_attention_blocks, ref_roi_n, C / num_attention_blocks]
         ref_x_new = ref_x_new.view(ref_roi_n, self.num_attention_blocks,
-                                   -1).permute(1, 0, 2)
+                                   num_C_per_attention_block).permute(1, 0, 2)
 
         # [roi_n, num_attention_blocks, C / num_attention_blocks]
         x_new = torch.bmm(weights, ref_x_new).permute(1, 0, 2).contiguous()
         # [roi_n, C]
-        x_new = self.fc(x_new.view(roi_n, -1))
+        x_new = self.fc(x_new.view(roi_n, num_C_per_attention_block))
```

Comment:
```suggestion
        x_new = self.fc(x_new.view(roi_n, C))
```
There is a bug in here. It should be `C` rather than `num_C_per_attention_block`